concepts/05 Widgets/Slider/00 Overview.md

```diff
@@ -6,28 +6,246 @@ The **Slider** is a widget that allows an end user to set a numeric value on a c
 
 The following code adds a simple **Slider** to your page. The **min** and **max** options limit the range of accepted values. The **value** sets the initial value for the widget.
 
+---
+##### jQuery
+
     <!--HTML-->
     <div id="sliderContainer"></div>
 
     <!--JavaScript-->
     $(function() {
         $("#sliderContainer").dxSlider({
-            min: 0, max: 100,
+            min: 0,
+            max: 100,
             value: 25
         });
     });
 
+##### Angular
+
+    <!--HTML-->
+    <dx-slider
+        [min]="minValue"
+        [max]="maxValue"
```

Comment:
It's unlikely that users will change min, max, or step. The only Slider option that needs binding to a component or state property is value. Please update all these examples.

Reply:
thx for your patience))